state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1966,7 +1966,14 @@ pub mod pallet {
 							output_address: T::AddressConverter::to_encoded_address(
 								output_address.clone(),
 							),
-							ccm_deposit_metadata: ccm_deposit_metadata.clone(),
+							ccm_deposit_metadata: cf_chains::CcmDepositMetadataEncoded {
```

Comment:
I think `CcmDepositMetadata` is also exposed via the CcmFailed event, so that one should be updated to use the encoded version.